sirepo/package_data/static/js/hellweg.js

```diff
@@ -404,10 +404,17 @@ SIREPO.app.directive('summaryTable', function() {
             function parseSummaryRows(summaryText) {
                 var text = summaryText.replace(/^(\n|.)*RESULTS\n+==+/, '').replace(/==+/, '');
                 var label = null;
+                var meanEnergy;
```

Comment:
Use `let meanEnergy`. In general, when we need to fiddle with older code we should update it with our current standards (within reason). So I would also use `const text...` and `let label...`